src/main/webapp/webrtc-test-tool.html

```diff
@@ -0,0 +1,1003 @@
+	var publish_packet_lost_info = document.getElementById("publish_packet_lost_info");
+
+	var publish_jitter_warn = document.getElementById("publish_jitter_warn");
+	var publish_jitter_info = document.getElementById("publish_jitter_info");
+
+
+
+	var round_trip_time_warn = document.getElementById("round_trip_time_warn");
+	var round_trip_time_info = document.getElementById("round_trip_time_info");
+
+	var source_resolution_warn = document.getElementById("source_resolution_warn");
+	var source_resolution_info = document.getElementById("source_resolution_info");
+
+	var ongoing_resolution_warn = document.getElementById("ongoing_resolution_warn");
+	var ongoing_resolution_info = document.getElementById("ongoing_resolution_info");
+
+	var on_going_fps_warn = document.getElementById("on_going_fps_warn");
+	var on_going_fps_info = document.getElementById("on_going_fps_info");
+
+	//These are for the play WebRTC Statistics information 
+
+	var play_min_bit_rate_warn = document.getElementById("play_min_bit_rate_warn");
+	var play_min_bit_rate_info = document.getElementById("play_min_bit_rate_info");
+
+	var play_average_bit_rate_warn = document.getElementById("play_average_bit_rate_warn");
+	var play_average_bit_rate_info = document.getElementById("play_average_bit_rate_info");
+
+	var play_max_bit_rate_warn = document.getElementById("play_max_bit_rate_warn");
+	var play_max_bit_rate_info = document.getElementById("play_max_bit_rate_info");
+
+	var play_packet_lost_warn = document.getElementById("play_packet_lost_warn");
+	var play_packet_lost_info = document.getElementById("play_packet_lost_info");
+
+	var play_jitter_warn = document.getElementById("play_jitter_warn");
+	var play_jitter_info = document.getElementById("play_jitter_info");
+
+	//var audio_level_warn = document.getElementById("audio_level_warn");
+	//var audio_level_info = document.getElementById("audio_level_info");
+
+
+
+	var incoming_resolution_warn = document.getElementById("incoming_resolution_warn");
+	var incoming_resolution_info = document.getElementById("incoming_resolution_info");
+
+	var frame_decoded_warn = document.getElementById("frame_decoded_warn");
+	var frame_decoded_infos = document.getElementById("frame_decoded_info");
+
+	var frame_dropped_warn = document.getElementById("frame_dropped_warn");
+	var frame_dropped_info = document.getElementById("frame_dropped_info");
+
+	var frame_received_warn = document.getElementById("frame_received_warn");
+	var frame_received_info = document.getElementById("frame_received_info");
+
+
+
+
+
+	var autoRepublishEnabled = true;
+	/**
+	 * Timer job that checks the WebRTC connection 
+	 */
+	 var autoRepublishIntervalJob = null;
+
+	 var streamId = "testStream2";
+
+	 var token = getUrlParameter("token");
+
+	 var webRTCPlayIntervalJob = null;
+
+	 var i = 0;
+
+	 var test_type;
+
+	 function startPublishing() {
+	 	test_type="publish";
+	 	publishWebRTCAdaptor.publish(streamId, token);
+	 }
+
+	 function startPlaying() {
```

Comment:
Play test is also a publish test. 
I think we should have just single button for testing, "Start WebRTC Testing"

We should provide both publish and play statistics at the same time. 

We should also provide some basic information about how WebRTC Testing works